core/src/Segmentation.cpp

```diff
@@ -14,6 +14,9 @@ Segmentation::Segmentation(fs::path path)
     if (metadata_.get<std::string>("type") != "seg") {
         throw std::runtime_error("File not of type: seg");
     }
+    if (metadata_.get<std::string>("vcps").empty()) {
```

Comment:
@stephenrparsons Actually, I think we need to revert this check. `vcps` is supposed to be empty when a Segmentation is created, and Segmentation:: hasPointSet() is used by VC to detect whether we can change the volume or draw a new segmentation curve when a Segmentation ID has been selected in the GUI.

Reply:
See #89